lib/utils/rruleUtil.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+let packageFound = false;
+try {
+  require('rrule');
+  packageFound = true;
+}
+catch (e) {
+  console.log('no package found');
+}
+
+let tryParseRRuleString = function(str) { return str;}
```

Comment:
why are functions defined as variables?

Reply:
to allow for multiple definitions with a single export, ill refactor

Reply:
you can use exactly the same `module.exports = {}` style with functions.